HRIS.Models/ClientProjectsDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace HRIS.Models;
+    public class ClientProjectsDto
+    {
+        public int Id { get; set; }
+        public int EmployeeId { get; set; }
+        public string? ClientName { get; set; }
+        public string? ProjectName { get; set; }
```

Comment:
Perhaps removing the ? from ClientName and ProjectName would be better because these fields should not ever be null?